frontend/src/pages/store/index.svelte

```diff
@@ -50,9 +232,18 @@
 			</div>
 		</div>
 	{/if}
-	{#await getStores()}
+	{#if userInfo && (userInfo.roles.includes('STAFF') || userInfo.roles.includes('STAFF'))}
```

Comment:
I'm confused as to why you check for the `STAFF` role twice here. 
Did you mean to check for `STAFF` and `ADMIN`?

I'm not sure but this might mean that depending on your configuration, some admins without a staff role won't be able to create stores.

Reply:
Yes, it was meant to be staff

Reply:
?